### PR TITLE
231 ability to change subscription endpoint

### PR DESCRIPTION
**Pull request by pawelszymczyk**

### CODE REVIEW

File: hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/SubscriptionEndpoint.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package pl.allegro.tech.hermes.api.endpoints;
 
+import pl.allegro.tech.hermes.api.EndpointAddress;
```
Comment (druminski) [inline hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/SubscriptionEndpoint.java]: unnecessary imports

File: hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/SubscriptionsEndpoint.java [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import com.wordnik.swagger.annotations.ApiOperation;
 import org.hibernate.validator.constraints.NotEmpty;
 import org.springframework.beans.factory.annotation.Autowired;
+import pl.allegro.tech.hermes.api.EndpointAddress;
```
Comment (druminski) [inline hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/SubscriptionsEndpoint.java]: same here, please optimize them

**Comment by druminski** [comment]
beside unused imports looks good to me :+1:

File: hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/SubscriptionEndpoint.java (hunk 1 above)
Comment (pawelszymczyk) [inline hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/SubscriptionEndpoint.java]: done

File: hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/SubscriptionsEndpoint.java (hunk 2 above)
Comment (pawelszymczyk) [inline hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/SubscriptionsEndpoint.java]: done

**Comment by adamdubiel** [comment]
:+1:

**Pull request merged**